yb-voyager/src/tgtdb/yugabytedb.go

```diff
@@ -774,3 +777,180 @@ func (yb *TargetYugabyteDB) recordEntryInDB(tx pgx.Tx, batch Batch, rowsAffected
 	}
 	return nil
 }
+
+func (yb *TargetYugabyteDB) GetDebeziumValueConverterSuite(snapshotMode bool) map[string]ConverterFn {
+	converterValueSuite := make(map[string]ConverterFn)
+	converterValueSuite["io.debezium.time.Date"] = func(columnValue string, formatIfRequired bool) (string, error) {
```

Comment:
I will skip reviewing individual converter functions and trust Aneesh's opinion on this.